src/components/MapGuide/index.tsx

```diff
@@ -49,9 +51,27 @@ const StyledMapGuideContext: React.FC = () => {
 };
 
 export const MapGuide: React.FC = () => {
+    const [isOpen, setIsOpen] = useState(false);
+    const [type, setType] = useState('none');
+
+    //checks type of action for mobile usage and only changes display if it is the same as last time
+    const changeTooltipDisplay = (value: boolean, how: string) => {
+        if (type === how) {
+            setIsOpen(value);
+            setType(how);
+        } else if (type === 'none') {
+            setIsOpen(value);
+            setType(how);
+        }
+    };
+
     return (
         <StyledMapGuideButton>
             <StyledMapGuideTooltip
+                onClick={() => changeTooltipDisplay(!isOpen, 'click')}
+                onOpen={() => changeTooltipDisplay(true, 'hover')}
+                onClose={() => changeTooltipDisplay(false, 'hover')}
+                open={isOpen}
                 arrow
```

Comment:
You can add `leaveTouchDelay`property that allows you to set the time after which the Tooltip will hide on touch screens. Default value is 1500